apps/web-app/src/components/PageContainer.tsx

```diff
@@ -27,25 +27,38 @@ export default function PageContainer({
 
     return (
         <>
-            <HStack align="center" justify="right" p="2">
+            <HStack align="center" justify="space-between" p="2">
                 <Link
```

Comment:
To improve UX, could you use the Nextjs link instead of the Chakra link. For that you would import `import NextLink from "next/link"` and use this instead:

```tsx
                <NextLink
                    href="/"
                >
                    Feedback
                </NextLink>
```

Reply:
okay i will do that.